Modules/_zstd/decompressor.c

```diff
@@ -61,7 +65,6 @@ _get_DDict(ZstdDict *self)
         return self->d_dict;
```

Comment:
Up to you, but I'd consider adding something like a `assert(PyMutex_IsLocked(&self->lock));` at the beginning of `_get_DDict`